app/src/main/java/org/mozilla/tv/firefox/MainActivity.kt

```diff
@@ -158,25 +157,23 @@ class MainActivity : LocaleAwareAppCompatActivity(), OnUrlEnteredListener, Media
         // See #1881 for details
         if (!safeIntent.hasExtra("TURBO_MODE")) {
```

Comment:
nit: can we put the onboarding code in a helper method? It's starting to get long and complicated. fwiw, I generally try to put non-trivial calls in `onCReate`/`onResume`/etc. into helper functions to make them read like high level methods:
```kotlin
initCrashReporting()
initTelemetry()
// etc.
```